tests/e2e/specs/shopper/checkout-form-warnings.test.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * Internal dependencies
+ */
+import { shopper } from '../../../utils';
+import {
+	SIMPLE_PRODUCT_NAME,
+	CUSTOMER_USERNAME,
+	CUSTOMER_PASSWORD,
+} from '../../../utils/constants';
+
+const block = {
+	name: 'Checkout',
+};
+
+if ( process.env.WOOCOMMERCE_BLOCKS_PHASE < 2 )
+	// eslint-disable-next-line jest/no-focused-tests
+	test.only( `skipping ${ block.name } tests`, () => {} );
+
+describe( 'Shopper → Checkout → Can see warnings when form is incomplete', () => {
+	beforeAll( async () => {
+		const isShopperLoggedIn = await shopper.isLoggedIn();
+
+		// @todo Find a better way to reset the checkout form instead of login then logout
+		if ( isShopperLoggedIn ) {
+			// Click on the "Log out" link in "My account" page
+			await await page.click(
+				'.woocommerce-MyAccount-navigation-link--customer-logout a'
+			);
+		} else {
+			await shopper.loginFromMyAccountPage(
+				CUSTOMER_USERNAME,
+				CUSTOMER_PASSWORD
+			);
+
+			await await page.click(
+				'.woocommerce-MyAccount-navigation-link--customer-logout a'
+			);
+		}
+	} );
```

Comment:
Why are you checking if a user is logged in, for you to log him out, or log him in again? What's the end goal here?

Reply:
We log him out to reset the checkout form by trigging a new session. The other method available to us is to clear each input manually, but this seemed to be slower as opposed to simply logout.

The automation bot created an issue regarding this particular case: #6123

Reply:
Thank you @senadir and @tarhi-saad for raising this question. In the recent [issue on which I'm working](https://github.com/woocommerce/woocommerce-gutenberg-products-block/issues/5780), I'm using the following code to logout user and navigate to checkout block to check if login option is available or not. 

We may also use the same code to logout and clear form fields:


```
await shopper.logout();
await shopper.goToShop();
await shopper.addToCartFromShopPage( SIMPLE_PRODUCT_NAME );
await shopper.block.goToCheckout();
```

Reply:
Thank you, @tarunvijwani, for suggesting an alternative way to achieve the same behavior. 🙌 In this case, since we are already on the `My account` page before login out, using the `shopper.logout` function will navigate us to the same page a second time [see implementation](https://github.com/woocommerce/woocommerce/blob/trunk/packages/js/e2e-utils/src/flows/shopper.js#L324).
So, to avoid unnecessary actions, we are directly clicking on the `logout` link instead.